resip/recon/UserAgent.cxx

```diff
@@ -89,6 +89,9 @@ UserAgent::UserAgent(ConversationManager* conversationManager, SharedPtr<UserAge
    mDum.addClientSubscriptionHandler("refer", mConversationManager);
    mDum.addServerSubscriptionHandler("refer", mConversationManager);
 
+   mDum.setServerPagerMessageHandler(mConversationManager);
+   mDum.setClientPagerMessageHandler(mConversationManager);
```

Comment:
instead of using mConversationManager here, it could be some other object for managing IM sessions